src/StaticFAQ/CustomBreadcrumbs.js

```diff
@@ -32,6 +32,18 @@ const CustomBreadcrumbs = ({ breadcrumbs, loginToken }: Props) => {
   const previousCategory = [...breadcrumbs].slice(-2)[0];
   const id = previousCategory && previousCategory.id;
 
+  const breadcrubsLength = breadcrumbs.map(b => b.title).join().length;
+
+  if (breadcrubsLength > 54) {
```

Comment:
I tried to find the maximum number of characters that fits onto screen.. I will play around a bit more now after #447 is merged. 😉

Reply:
btw I would advice to use constant, instead of "magic" numbers

`if (breadcrubsLength > maxBreadcrumbLength) {` looks far better & is more readable than `if (breadcrubsLength > 54) {`

Reply:
@jalbertsr the only problem now is with mobile, should I make a mobile and desktop version?

Reply:
let's ask @mjancik